data/en/fileupload.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fileUpload",
   "type": "function",
-  "syntax": "fileUpload(string destination,[string fileField,[string accept,[string nameConflict]]])",
+  "syntax": "fileUpload(string destination,[string fileField,[string accept,[string nameConflict,[boolean strict]]]])",
```

Comment:
We do not use data type in `syntax`. Could be removed.

Reply:
@KrunchMuffin contributed at 10th November for the last time until now, so I don't know, if he will come back...